frontend/compositions/header/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react"
```

Comment:
Could we rename this component to be more specific (maybe `DashboardHeader`?) To differentiate it from other "headers" we have throughout the app

Reply:
Yes, will rename!